client/src/banners/active-banner.tsx

```diff
@@ -82,45 +87,28 @@ function Banner(props: BannerProps) {
   );
 }
 
-function DeveloperNeedsBanner({ onDismissed }: { onDismissed: () => void }) {
+function CommonSurveyBanner({ onDismissed }: { onDismissed: () => void }) {
+  const ga = useGA();
+
   return (
     <Banner
-      id={DEVELOPER_NEEDS_ID}
-      classname="developer-needs"
-      title={"MDN Survey"}
+      id={COMMON_SURVEY_ID}
+      title={"Learning web development survey"}
       copy={
-        "Help us understand the top 10 needs of Web developers and designers."
+        "Help us understand how to make MDN better for beginners (5 minute survey)"
       }
       cta={"Take the survey"}
-      url={"https://qsurvey.mozilla.com/s3/Developer-Needs-Assessment-2019"}
+      url={"https://www.surveygizmo.com/s3/6175365/59cfad9c04cf"}
```

Comment:
Nit: there's no need for curly brackets with it's just a string. For example...
```suggestion
      url="https://www.surveygizmo.com/s3/6175365/59cfad9c04cf"
```